Shared/Samples/Apply scheduled updates to preplanned map area/README.metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+    "category": "Edit and Manage Data",
```

Comment:
I noticed that the old iOS sample was in the "Maps" category. This change makes sense to me, just making sure it was intended.